README.md

```diff
@@ -1,25 +1,224 @@
-# Terraform zabbix provider
+Terraform Provider
```

Comment:
```suggestion
Terraform Provider for Zabbix
```
See:
- https://github.com/terraform-providers/terraform-provider-google/blob/master/README.md#terraform-provider-for-google-cloud-platform
- https://github.com/terraform-providers/terraform-provider-helm/blob/master/README.md#terraform-provider-for-helm
- https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/README.md#terraform-provider-for-azure-resource-manager

Reply:
No luck ! I took pagerduty and datadog as example ^^ 

https://github.com/terraform-providers/terraform-provider-datadog
https://github.com/terraform-providers/terraform-provider-pagerduty